### DISCUSSION TITLE
Nouns Builder L2 Testnets Feedback

### DISCUSSION THREAD

**Discussion by isaaczaak** [ourzora/nouns-builder#297] [Ideas]
Thanks for testing and taking Nouns Builder testnets for a spin!

**How to share feedback**
1. What issues did you experience during testing?
2. What should have been the expected outcome/behaviour?
3. Any further notes

Where to test:
[Nouns Builder Testnet](https://testnet.nouns.build)

**Comment by Valcoholics** [comment]
GM, 

Overall, it's looking good, but I did run into small issues. I tried to do a few transactions like an edition proposal, settling auctions, and bidding and saw consistent gas prices throughout which was good. When I tried adding a transaction to send ETH to an ENS name, it didn't allow me to do it that way. However, using a regular 0xaddress worked like a charm. I also attempted to change the artwork through a proposal, but unfortunately, I couldn't upload the new folder or download the demo folder It seems there's a little hiccup in that area. I tried a custom transaction to change the vote delay and vote period, and no issues there. All of the params loaded and really fast. Also I noticed is how fast proposals get posted after submitting. That's definitely a thumbs up!

https://vimeo.com/842743329

24hr edit: More feedback, now voting on those proposals but the vote is not submitting.

**Reply by neokry** [reply]
thanks for flagging @Valcoholics should be fixed with #303
